### PR TITLE
[dch] Pull Request to review / add dch stat block template

### PR DESCRIPTION
**Pull request by CBairdAsh**
## ADD :
	- Display/dch/character-dch.html.twig : Created a macro twig, base character sheet with extension sheet all for the character sheet template. Made sure to extend from generic base twig.
	- Display/dch/dch-base.html.twig : Created a macro twig, base character sheet with extension sheet all for the character sheet template. Made sure to extend from generic base twig.
	- Display/dch/dch-macros.html.twig : Created a macro twig, base character sheet with extension sheet all for the character sheet template. Made sure to extend from generic base twig.
	- css/dch/dch.css : Updated CSS : style update to match wa. Adjusted styles to match World Anvil's requirements. added 'dch' prefix onto class definitions. Encapsualted all styles by system-dch. removed attr_title and attr_bench in favor of a compound nth-child assignment that was more flexible and worked better with macro creation in twig.
    - Schema/dch/character-dch.yml : Updated the yaml with unique_reference and display_template values
    ## MOD :
    - Display/dch/test_character.html : updated ux layout prototype to reflect updated css

### CODE REVIEW

File: Display/dch/dch-base.html.twig [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{% extends 'RPGSRDBundle:Templates:base.html.twig' %}
+{% block body %}
+  <div class="dch_page">
+    <div class="row stripe_hightlight mb-2em">
```
Comment (GorkamWorka) [inline Display/dch/dch-base.html.twig]: mb is bootstrap 4 iirc, WA is only bootstrap 3

File: Display/dch/dch-macros.html.twig [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+  <div class="col-md-4">
+    <div class="row">
+      <div class="col-md-6 attr_title">{{atr_title}}</div>
+      <div class="col-md-6">{{ atrib }}</div>
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro benchmrk_ux(atr_val, atr_title) %}
+  <div class="row mb-1em">
```
Comment (GorkamWorka) [inline Display/dch/dch-macros.html.twig]: same comment about mb

File: Display/dch/dch-macros.html.twig [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+    <div class="col-md-3">{{ atr_title }}</div>
+    <div class="col-md-9">{{ atr_val }}</div>
+  </div>
+{% endmacro %}
+
+{% macro stripe_it(guts) %}
+  <div class="row stripe_hightlight mb-1em cat-bar">
```
Comment (GorkamWorka) [inline Display/dch/dch-macros.html.twig]: mb

File: css/dch/dch.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.system-dch .dch_container-fluid .row:nth-child(1),
+.system-dch .dch_container-fluid .row:nth-child(n+2) div:nth-child(1),
+.system-dch .attr_title {
+  font-weight: 700;
+  font-style: italic;
+}
+
+.system-dch .dch_container-fluid .row:nth-child(n+2) div:nth-child(2) {
```
Comment (GorkamWorka) [inline css/dch/dch.css]: I'm not sure this will compute on WA.
Please be advise that WA do not use any css pre-compiler ( less or any other )

Comment (CBairdAsh) [reply css/dch/dch.css]: nth-child is processed by the browser itself. It's part of CSS3 specification. It's not something that is intended to be processed by php. For reference: https://www.w3schools.com/cssref/sel_nth-child.asp

File: Display/dch/dch-base.html.twig (hunk 1 above)
Comment (CBairdAsh) [reply Display/dch/dch-base.html.twig]: Actually no. 'mb-2em' is just me abbreviating 'margin-bottom' CSS command

File: Display/dch/character-dch.html.twig [hunk 5]
```diff
@@ -0,0 +1,96 @@
+{% extends 'RPGSRDBundle:Templates:dch/dch-base.html.twig' %}
+{% import 'RPGSRDBundle:Templates:dch/dch-macros.html.twig' as macros %}
+{% block dch_attributes %}
+<div class="dch_container-fluid mb-2em">
```
[the remark this answers is not in the document]
Comment (CBairdAsh) [reply Display/dch/character-dch.html.twig]: I used Bootstrap 3 pulled from their cdn..

File: Display/dch/dch-macros.html.twig (hunk 3 above)
Comment (CBairdAsh) [reply Display/dch/dch-macros.html.twig]: Actually no. 'mb-2em' is just me abbreviating 'margin-bottom' CSS command

File: Display/dch/dch-macros.html.twig (hunk 2 above)
Comment (CBairdAsh) [reply Display/dch/dch-macros.html.twig]: Actually no. 'mb-2em' is just me abbreviating 'margin-bottom' CSS command

**Comment by iamromeo** [comment]
@CBairdAsh will be handling this within the next two days! Thank you so very much

**Pull request merged**